diff --git a/Makefile.libretro b/Makefile.libretro
index 5b90d40..1ec93b5 100644
--- a/Makefile.libretro
+++ b/Makefile.libretro
@@ -451,6 +451,15 @@ else ifeq ($(platform), retrofw)
 	PLATFORM_DEFINES += -ffast-math -march=mips32 -mtune=mips32 -mhard-float -fomit-frame-pointer
 
 # Miyoo
+else ifeq ($(platform), trimui)
+	TARGET := $(TARGET_NAME)_libretro.so
+	CC = $(CROSS_COMPILE)gcc
+	CXX = $(CROSS_COMPILE)g++
+	AR = $(CROSS_COMPILE)ar
+	fpic := -fPIC
+	SHARED := -shared -Wl,--version-script=src/drivers/libretro/link.T -Wl,-no-undefined
+	PLATFORM_DEFINES += -fomit-frame-pointer -ffast-math -mcpu=arm926ej-s -O3
+
 else ifeq ($(platform), miyoo)
 	TARGET := $(TARGET_NAME)_libretro.so
 	CC = /opt/miyoo/usr/bin/arm-linux-gcc
