diff --git a/Makefile b/Makefile
index f5e3bf2..6764025 100644
--- a/Makefile
+++ b/Makefile
@@ -536,6 +536,21 @@ else ifeq ($(platform), miyoomini)
 	ARCH = arm
 	BUILTIN_GPU = neon
 
+# TRIMUI MODEL S
+else ifeq ($(platform), trimui)
+  TARGET := $(TARGET_NAME)_libretro.so
+  CC  = $(CROSS_COMPILE)gcc
+  CXX = $(CROSS_COMPILE)g++
+  AR  = $(CROSS_COMPILE)ar
+  fpic := -fPIC -DPIC
+  SHARED := -shared -nostdlib -Wl,-version-script=link.T
+  CFLAGS += -fomit-frame-pointer -ffast-math -march=armv5te -mtune=arm926ej-s
+  CFLAGS += -DSMALL_TRANSLATION_CACHE
+  CFLAGS += -flto
+  LDFLAGS += -flto
+  HAVE_DYNAREC := 1
+  CPU_ARCH := arm
+  
 # Windows
 else
 	TARGET := $(TARGET_NAME)_libretro.dll
diff --git a/gpsp_config.h b/gpsp_config.h
index 625f46e..99f4780 100644
--- a/gpsp_config.h
+++ b/gpsp_config.h
@@ -13,8 +13,8 @@
 
 /* Cache sizes and their config knobs */
 #if defined(SMALL_TRANSLATION_CACHE)
-  #define ROM_TRANSLATION_CACHE_SIZE (1024 * 1024 * 2)
-  #define RAM_TRANSLATION_CACHE_SIZE (1024 * 384)
+  #define ROM_TRANSLATION_CACHE_SIZE (1024 * 1024 * 4)
+  #define RAM_TRANSLATION_CACHE_SIZE (1024 * 512)
 #else
   #define ROM_TRANSLATION_CACHE_SIZE (1024 * 1024 * 10)
   #define RAM_TRANSLATION_CACHE_SIZE (1024 * 512)
